mage_ai/frontend/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "export": "next build && next export",
-    "export_prod": "next build && next export -o ../server/frontend_dist",
-    "export_prod_base_path": "next build && next export -o ../server/frontend_dist_base_path_template",
+    "export": "next build",
+    "export_prod": "next build && mv out ../server/frontend_dist",
+    "export_prod_base_path": "next build && mv out ../server/frontend_dist_base_path_template",
```

Comment:
This needs to be updated as well. This should work:
```
"export_prod_base_path": "next build && rm -rf ../server/frontend_dist_base_path_template && mv frontend_dist ../server/frontend_dist_base_path_template",
```

Reply:
fixed db83522c4